lib/blockscore.rb

```diff
@@ -1,5 +1,38 @@
-require "httparty"
+require 'forwardable'
+require 'httparty'
+require 'json'
+require 'ostruct'
+require 'uri'
 
-Dir[File.dirname(__FILE__) + '/blockscore/*.rb'].each do |file|
-  require file
-end
\ No newline at end of file
+require 'blockscore/actions/all'
+require 'blockscore/actions/create'
+require 'blockscore/actions/delete'
+require 'blockscore/actions/retrieve'
+require 'blockscore/actions/update'
+require 'blockscore/base'
+require 'blockscore/candidate'
```

Comment:
I think it would be better to sort these by concern rather than alphabetically. Actions, Resources, Errors, etc